commands/device/list-devices.ts

```diff
@@ -19,18 +17,20 @@ export class ListDevicesCommand implements ICommand {
 			this.$devicesServices.initialize({platform: args[0], deviceId: null, skipInferPlatform: true}).wait();
 
 			let table: any = createTable(["#", "Device Name", "Platform", "Device Identifier"], []);
-			let action: (device: Mobile.IDevice) => IFuture<void>;
+			let action: (_device: Mobile.IDevice) => IFuture<void>;
 			if (this.$options.json) {
 				this.$logger.setLevel("ERROR");
 				action = (device) => {
-					return (() => { this.$logger.out(JSON.stringify({
-						identifier: device.deviceInfo.identifier,
-						platform: device.deviceInfo.platform,
-						model: device.deviceInfo.model,
-						name: device.deviceInfo.displayName,
-						version: device.deviceInfo.version,
-						vendor: device.deviceInfo.vendor
-					}))}).future<void>()();
+					return (() => { 
+						this.$logger.out(JSON.stringify({
+							identifier: device.deviceInfo.identifier,
+							platform: device.deviceInfo.platform,
+							model: device.deviceInfo.model,
+							name: device.deviceInfo.displayName,
+							version: device.deviceInfo.version,
+							vendor: device.deviceInfo.vendor
+						}));
```

Comment:
I think we can simplify this code. Just `this.$logger.out(JSON.stringify(device.deviceInfo))` :smile_cat: